orchagent/pfcactionhandler.h

```diff
@@ -39,15 +47,28 @@ class PfcWdActionHandler
         }
 
         static void initWdCounters(shared_ptr<Table> countersTable, const string &queueIdStr);
+        void initCounters(void);
```

Comment:
There are initWdCounters and initCounters in this class, which might be a little difficult to understand. Consider rename to be more specific?

Reply:
Same as above